src/server/protocol.ts

```diff
@@ -1742,15 +1744,13 @@ namespace ts.server.protocol {
          */
         prefix?: string;
         /**
-         * If enabled, TypeScript will search through all external modules' exports and add them to the completions list.
-         * This affects lone identifier completions but not completions on the right hand side of `obj.`.
+         * @deprecated Use Options
          */
-        includeExternalModuleExports: boolean;
+        includeExternalModuleExports?: boolean;
         /**
-         * If enabled, the completion list will include completions with invalid identifier names.
-         * For those entries, The `insertText` and `replacementSpan` properties will be set to change from `.x` property access to `["x"]`.
+         * @deprecated Use Options
          */
-        includeInsertTextCompletions: boolean;
+        includeInsertTextCompletions?: boolean;

```

Comment:
Looks like we have two separate names for the same concept (i.e. different thing in protocol vs services)

Reply:
This is a deprecated property of `CompletionsRequestArgs` to support the old name, the new name is later on in this file.